src/modules/IscsiClientLib.rb

```diff
@@ -946,6 +954,8 @@ def autoLogOn
         result = SCR.Execute(path(".target.bash_output"), GetAdmCmd("-m fw -l"))
         ret = false if result["exit"] != 0
         log.info "Autologin into iBFT : #{result}"
+        result = SCR.Execute(path(".target.bash_output"), GetAdmCmd("-m discovery -t fw"))
```

Comment:
The `result` seems to be unused (besides logging it). Is that OK?

Reply:
I was unsure here. As the command has no immediate result that must be used I think it's fair not to add additional failure checks here. It's not making things worse **not** to check here while it might introduce additional failures that might be wrong **if** I check.

Reply:
In general this whole patch is only as far as I understood the iscsi tools. Which might be totally off in any case...